### DISCUSSION TITLE
Use 4 Channels of a H801 for 2 CCT-Lights

### DISCUSSION THREAD

**Discussion by Andreas-Fink** [arendst/Tasmota#11650] [Q&A] [answered]
Hi Together,

is there any chance to use one Tasmota-flashed H801 (5 Channels) for 2 CCT-Lights. Maybe by using Wc and Ww for the first stripe and R and G (or so on) as Wc and Ww for the other Stripe? With virtually 2 lights like in the CT/RGB-Split-Mode.

Regards,
Andreas

**Comment by barbudor** [comment, marked as answer]
Tasmota can manage only 1 "light"
However with setoption68 you can split the PWM channels individually. That will allow you to control each separately. 
Up to you to recreate the "light" object into your home automation by driving the PWM directly

**Reply by sfromis** [reply]
If you split into RGB and CT channels using [ChannelRemap](https://tasmota.github.io/docs/Commands/#channelremap) you could indeed control two of the RGB colors to work as a CCT light, which you then can encapsulate in your automation setup. Tasmota will of course still se it as RGB, but that does not stop you from hooking up differently.

**Reply by Andreas-Fink** [reply]
I think I will play a bit with the two options. In fact I can simulate the wished behavior with OpenHAB both ways, but it would be a bit more „dirty“ and I would prefere a more „clean“ way. But if there is no other way, than it needs to ok that way. Maybe Tasmota will learn to handle two separate CCT-Light in the future, like the splitted RGB+CCT-Lights.
